packages/image_picker/image_picker_for_web/lib/image_picker_for_web.dart

```diff
@@ -320,6 +320,9 @@ class ImagePickerPlugin extends ImagePickerPlatform {
     // TODO(dit): Reimplement this with the showPicker() API, https://github.com/flutter/flutter/issues/130365
     element.click();
   }
+
+  /// Removes the file input element
+  void _remove(html.Element element) => _target.children.clear();
```

Comment:
This `_remove(element)` is not removing element, it's clearing _target.children.